e2e/cypress/integration/05-mmis/01-mmis-basics.cy.js

```diff
@@ -286,11 +313,8 @@ describe('MMIS Basics', { tags: ['@apd', '@default', '@mmis'] }, function () {
         'activity-proposed-solution-field',
         `<p>${mmisBasics.activities[0].activityOverview.proposedSolution}</p>`
       );
-    });
-
-    it('tests Key State Personnel page', function () {
-      const mmisBasics = this.mmisBasics;
 
+      // Key State Personnel
```

Comment:
Hmm, personal preference, but I think I liked the way it was structured by page because there's so much to test on each page. I'm curious about your thoughts on this!